beta-prompts/beta_prompts/seaborn_analytics.py

```
@@ -0,0 +1,1413 @@
+        sns.regplot(
+            data=data_sorted,
+            x=list(range(len(data_sorted))),
+            y="quality_score",
+            scatter=False,
+            color=SEABORN_DARK_THEME["neon_lime"],
+            line_kws={"linewidth": 4, "alpha": 0.8},
+            ax=ax,
+        )
+
+        # Calculate and display trend statistics
+        from scipy.stats import linregress
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)